client/src/app/[locale]/layout.tsx

```diff
@@ -0,0 +1,36 @@
+import "./globals.css";
+
+import { getTranslations } from "next-intl/server";
+
+export async function generateMetadata({ params: { locale } }: { params: { locale: string } }) {
+  const t = await getTranslations({ locale });
+
+  return {
+    title: t("Rangelands Data Platform"),
+    description: t(
+      "Diverse ecosystems crucial for both wildlife and people. Explore their beauty and significance with our Vital Ecosystem Atlas, advocating for their protection and restoration.",
```

Comment:
next-intl doesn't support keys that contain dots: https://next-intl-docs.vercel.app/docs/usage/messages#messages-other-styles. This means that we'll never be able to translate this string.

To avoid this issue, I'm proposing the following solution.

In the `i18n.ts` file, add the following code:
```ts
import { getTranslations as getNextIntlTranslations, useTranslations as useNextIntlTranslations } from "next-intl";

export const getTranslations = async ({ locale }: { locale: string }) => {
  const t = await getNextIntlTranslations({ locale });
  return (str: string) => t(str.replaceAll(".", "{{dot}}"));
};

export const useTranslations = () => {
  const t = useNextIntlTranslations();
  return (str: string) => t(str.replaceAll(".", "{{dot}}"));
};
```

In the same file, replace:
```ts
await tx.fetchTranslations(locale, { refresh: true });
const translations = tx.cache.getTranslations(locale);
```
by:
```ts
await tx.fetchTranslations(locale, { refresh: true });
const translations = Object.fromEntries(Object.entries(tx.cache.getTranslations(locale)).map(([key, value]) => [key.replaceAll('.', '{{dot}}'), value]));
```

Now everywhere `getTranslations` and `useTranslations` are called, use the functions from `i18n.ts` instead.

What this code does is that it makes next-intl think that keys such as `Hello.` are `Hello{{dot}}`, while letting Transifex properly extract the correct strings.